api/lib/domain/usecases/modify-localized-challenge.js

```diff
@@ -11,6 +12,11 @@ export async function modifyLocalizedChallenge({ isAdmin, localizedChallenge },
     if (!isAdmin && localizedChallenge.status !== originalLocalizedChallenge.status) {
       throw new ForbiddenError();
     }
+    if (localizedChallenge.status === LocalizedChallenge.STATUSES.PLAY && originalLocalizedChallenge.status !== localizedChallenge.status) {
+      localizedChallenge.validatedAt = new Date();
+    } else {
+      localizedChallenge.validatedAt = originalLocalizedChallenge.validatedAt;
+    }
```

Comment:
Cette manière de faire semble un peu fragile (on a l’impression qu’on ne maitrise pas les champs qui sont mis à jour), ce serait peut-être plus sécurisant de se faire une méthode update sur le model comme dans d’autres usecases :

```suggestion
    originalLocalizedChallenge.update(localizedChallenge);
```

Reply:
Pas obligé.

Reply:
oui je vois, will do